### DISCUSSION TITLE
upload backup web api issue

### DISCUSSION THREAD

**Discussion by ashok-cse** [pocketbase/pocketbase#3511] [Q&A] [answered]
Endpoint: /api/backups/upload

`    const file = e.target.files[0]

   const formData = new FormData()

   formData.append('file', file)

	try {
		const response = await fetch(`${endpoint}/api/backups/upload`, {
			method: 'POST',
			headers: {
				Authorization: token
			},
            body: formData
		})

        const resdata = await response.json()

        console.log('data', resdata);

	} catch (error) {}
}`

getting 404 response, please guide me further if there any issue with my approach

**Comment by ganigeorgiev** [comment, marked as answer]
What version of PocketBase do you use (_the upload endpoint was added in v0.18.0; you can find your version in the bottom-right corner of the Admin UI_)?

Note that you can also use directly the JS SDK:

```js

await pb.backups.upload({ file: yourFile });
```

**Reply by ashok-cse** [reply]
currently, i am using PocketBase version 0.16.5

can i use this JS SDK with sveltekit?

**Reply by ganigeorgiev** [reply]
Yes. It is a plain JS library without any external dependencies (it relies only on native apis like `fetch` and `EventSource`).

Make sure to check the [JS SDK README](https://github.com/pocketbase/js-sdk) for more details.

**Reply by ashok-cse** [reply]
Thank you! 🤩
